zeek

```diff
@@ -181,10 +194,65 @@ main() {
 		else
 			echo "Zeek is already stopped." >&2
 		fi
-		
+
 		$SUDO docker rm --force "$container" >/dev/null 2>&1
 		;;
 
+	readpcap)
+		#Command(s) needed to process a pcap file
+
+		if [ "$running" = "true" ]; then
+			echo "Zeek is already running, exiting." >&2
+			exit 0
+		fi
+
+		init_zeek_cfg
+
+		# create the volumes required for peristing user-installed zkg packages
+		$SUDO docker volume create zeek-zkg-script >/dev/null
+		$SUDO docker volume create zeek-zkg-plugin >/dev/null
+		$SUDO docker volume create zeek-zkg-state >/dev/null
+
+		docker_cmd=("docker" "run" "--rm")          # start container in the foreground
+		docker_cmd+=("--workdir" "/usr/local/zeek/logs/")
+
+		# allow packages installed via zkg to persist across restarts
+		docker_cmd+=("--mount" "source=zeek-zkg-script,destination=/usr/local/zeek/share/zeek/site/packages/,type=volume")
+		docker_cmd+=("--mount" "source=zeek-zkg-plugin,destination=/usr/local/zeek/lib/zeek/plugins/packages/,type=volume")
+		docker_cmd+=("--mount" "source=zeek-zkg-state,destination=/root/.zkg,type=volume")
+
+		# mirror the host timezone settings to the container
+		docker_cmd+=("--mount" "source=/etc/localtime,destination=/etc/localtime,type=bind,readonly")
+
+		# persist and allow accessing the logs from the host
+		docker_cmd+=("--mount" "source=$HOST_ZEEK/manual-logs,destination=/usr/local/zeek/logs/,type=bind")
```

Comment:
Optional: To give the user the ability to specify the log directory, you'd replace `$HOST_ZEEK/manual-logs` with the value from the user. Potentially something like this:
```bash
# this line would go where arguments are processed
MANUAL_LOG_DIR=$(realpath "${2:-$HOST_ZEEK/manual-logs}")

docker_cmd+=("--mount" "source=$MANUAL_LOG_DIR,destination=/usr/local/zeek/logs/,type=bind")
```

Reply:
Marvelous!  Added, and a quick note in the help text.

Reply:
Checked that this honors the command line param if specified, and falls back on /opt/zeek/manual-logs if not.